packages/cli/src/commands/generate/config/generate_config_command.test.ts

```diff
@@ -107,71 +107,98 @@ void describe('generate config command', () => {
       'config --branch branch_name --app-id app_id --out-dir /foo/bar --format mjs'
     );
     assert.equal(generateClientConfigMock.mock.callCount(), 1);
-    assert.deepStrictEqual(generateClientConfigMock.mock.calls[0].arguments, [
+    assert.deepStrictEqual(
+      generateClientConfigMock.mock.calls[0].arguments[0],
       {
-        name: 'branch_name',
         namespace: 'app_id',
+        name: 'branch_name',
         type: 'branch',
-      },
-      '/foo/bar',
-      ClientConfigFormat.MJS,
-    ]);
+      }
+    );
+    assert.strictEqual(
+      generateClientConfigMock.mock.calls[0].arguments[1],
+      '/foo/bar'
+    );
+    assert.strictEqual(
+      generateClientConfigMock.mock.calls[0].arguments[2],
+      ClientConfigFormat.MJS
+    );
   });
 
   void it('can generate to custom absolute path', async () => {
     await commandRunner.runCommand(
       'config --stack stack_name --out-dir /foo/bar --format ts'
     );
     assert.equal(generateClientConfigMock.mock.callCount(), 1);
-    assert.deepStrictEqual(generateClientConfigMock.mock.calls[0].arguments, [
-      {
-        stackName: 'stack_name',
-      },
-      '/foo/bar',
-      ClientConfigFormat.TS,
-    ]);
+    assert.deepStrictEqual(
+      generateClientConfigMock.mock.calls[0].arguments[0],
+      { stackName: 'stack_name' }
+    );
+    assert.strictEqual(
+      generateClientConfigMock.mock.calls[0].arguments[1],
+      '/foo/bar'
+    );
+    assert.strictEqual(
+      generateClientConfigMock.mock.calls[0].arguments[2],
+      ClientConfigFormat.TS
+    );
   });
 
   void it('can generate to custom relative path', async () => {
     await commandRunner.runCommand(
       'config --stack stack_name --out-dir foo/bar --format mjs'
     );
     assert.equal(generateClientConfigMock.mock.callCount(), 1);
-    assert.deepStrictEqual(generateClientConfigMock.mock.calls[0].arguments, [
-      {
-        stackName: 'stack_name',
-      },
-      'foo/bar',
-      ClientConfigFormat.MJS,
-    ]);
+    assert.deepStrictEqual(
```

Comment:
why split this apart? The previous assertion seems more concise

Reply:
The way that this test suite is set up, there was way to match the last parameter here. The last parameter function is a anonymous function.

Reply:
we could assert on `args.slice(0, 3)` (or however many we want to check)